### PR TITLE
Add known issues for 64-bit ARM

### PR DESCRIPTION
**Pull request by SueChaplain**
Update known issues table in the 0.20.0 release notes.

[ci skip]

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
@SueChaplain can you please add https://github.com/eclipse/openj9/issues/9251 as a limitation as well.

**Comment by SueChaplain** [comment]
>can you please add #9251 as a limitation as well.

@pshipton 
I've updated the table but I'm not sure I have described the problem or workaround correctly, so please let me know if you want any changes.

File: doc/release-notes/0.20/0.20.md [hunk 1]
```diff
@@ -98,6 +98,49 @@ The v0.20.0 release contains the following known issues and limitations:
...
+<td valign="top">AOT compilation is not working by default</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">When using shared classes, startup performance is not optimized in this release. </td>
+<td valign="top">To turn on AOT explicitly, specify the <tt>-Xquickstart</tt> or <tt>-Xaot</tt> option.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/9251">#9251</a></td>
+<td valign="top">Problems reading system dumps</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">Although core files are successfully generated on AArch64 systems, the dump viewer (<tt>jdmpview</tt>) in an Aarch64 JVM is unable to read them.</td>
```
Comment (knn-k) [inline doc/release-notes/0.20/0.20.md:140]: I would like words like "in some cases" to be added here.  It does not always fail, depending on unknown condition.

Comment (SueChaplain) [reply doc/release-notes/0.20/0.20.md]: Done

**Comment by pshipton** [comment]
lgtm

**Pull request merged**